Lib/traceback.py

```diff
@@ -532,6 +503,47 @@ def __init__(self, exc_type, exc_value, exc_traceback, *, limit=None,
             self.msg = exc_value.msg
         if lookup_lines:
             self._load_lines()
+        self.__suppress_context__ = \
+            exc_value.__suppress_context__ if exc_value else False
+
+        # Convert __cause__ and __context__ to `TracebackExceptions`s, use a
+        # queue to avoid recursion
+        if not _is_chained:
+            queue = [(self, exc_value)]
+            while queue:
+                te, e = queue.pop()
+                # Gracefully handle (the way Python 2.4 and earlier did) the
+                # case of being called with no type or value (None, None, None)
```

Comment:
I know this comment was always there, but I think it has outlived its relevance. And AFAICT it really is just trying to explain that `e` may be None. I propose to just delete the comment.